### PR TITLE
Mise à jour de mes infos, création d'une ville Bayonne

### PR DESCRIPTION
**Pull request by Yharra**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -5898,7 +5898,7 @@ var data = {
         }
       ],
       "cities": [
-        "Paris"
+        "Paris, Pays basque"
```
Comment (dadoonet) [inline baggers.js:5901]: It should be here:

```
"cities": [
        "Paris",
        "Bayonne"
      ]
```

**Review by dadoonet — CHANGES_REQUESTED** [review]
Please also change the end of this file to add Bayonne:

```
  "cities": {
    "Aix-en-Provence": {
      "name": "Aix-en-Provence",
      "ville_img": "img/villes/BBL.jpg",
      "lat": 43.529742,
      "lng": 5.447427
    },

    "Avignon": {
      "name": "Avignon",
      "ville_img": "img/villes/BBL_Avignon.jpg",
      "lat": 43.941557,
      "lng": 4.8332525
    },
```

Make sure you respect the alphabetical order when adding Bayonne. Lat/Lon must be the center of Bayonne ideally.

And send another PR to add the image for Bayonne: https://github.com/brownbaglunch/BrownBagLunch/tree/gh-pages/img/villes

**Comment by dadoonet** [comment]
Ping?

**Comment by Yharra** [comment]
Bonjour, 

oui, mais j’ai pas eu le temps de m’en occuper !

**Comment by dadoonet** [comment]
Pas de News. Closing.

**Pull request closed without merging**